manageiq-operator/tools.go

```diff
@@ -0,0 +1,5 @@
+// +build tools
+
+// Place any runtime dependencies as imports in this file.
+// Go modules will be forced to download and install them.
+package tools
```

Comment:
Do we need this file?

Reply:
It is generated by `operator-sdk`, so I think we can just keep it as it is?

Reply:
If we're not using it for anything I'd rather not keep it around for no reason.

I'm not sure I even understand the comment. What runtime dependencies would we declare there that we haven't declared elsewhere?

Reply:
Done, also opened an issue at https://github.com/operator-framework/operator-sdk/issues/2828 to get more comments from operator-sdk experts.